salt/modules/win_dns_client.py

```diff
@@ -45,7 +45,10 @@ def get_dns_servers(interface='Local Area Connection'):
         for iface in c.Win32_NetworkAdapter(NetEnabled=True):
             if interface == iface.NetConnectionID:
                 iface_config = c.Win32_NetworkAdapterConfiguration(Index=iface.Index).pop()
-                return list(iface_config.DNSServerSearchOrder)
++                try:
```

Comment:
@cheng0919, it looks like you forgot to remove the `+` diff chars at the beginning of the lines.  Also, the `except` will need to be `except TypeError`.  Thanks for fixing this.